kubernetes/jobs/metrics-report-generator/cronjob.yaml

```diff
@@ -80,6 +102,26 @@ spec:
                     secretKeyRef:
                       name: documentdb
                       key: READONLY_PSW
+                - name: REPORTS_MAIL_RECIPIENTS
+                  valueFrom:
+                    secretKeyRef:
+                      name: documentdb
+                      key: REPORTS_MAIL_RECIPIENTS
+                - name: MAIL_SENDER
+                  valueFrom:
+                    secretKeyRef:
+                      name: documentdb
+                      key: MAIL_SENDER
+                - name: SMTP_USERNAME
+                  valueFrom:
+                    secretKeyRef:
+                      name: documentdb
+                      key: SMTP_USERNAME
+                - name: SMTP_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      name: documentdb
+                      key: SMTP_PASSWORD
```

Comment:
These values are not in the `documentdb` secret. 
We need to create a new K8s secret [here](https://github.com/pagopa/pdnd-interop-platform-deployment/blob/7ff3498ee8c43e24607e40f925134270543b05c7/.github/workflows/deploy-configs.yaml#L92), referencing the correct secrets names on AWS Secrets Manager
re